### PR TITLE
Use DarkSky for weather

### PR DESCRIPTION
**Pull request by amclelland**
No more Yahoo! :tada:

### CODE REVIEW

File: app/assets/javascripts/skycons.js [hunk 1]
```diff
@@ -0,0 +1,730 @@
+(function(global) {
```
Comment (artfuldodger) [inline app/assets/javascripts/skycons.js]: Should this be in vendor?

File: config/initializers/assets.rb [hunk 2]
```diff
@@ -9,3 +9,5 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+
+Rails.application.config.assets.precompile += %w( skycons.js )
```
Comment (artfuldodger) [inline config/initializers/assets.rb]: Shouldn't this just be required in `application.js`?

File: app/assets/javascripts/skycons.js (hunk 1 above)
Comment (amclelland) [inline app/assets/javascripts/skycons.js]: lol whoops yes.

File: config/initializers/assets.rb (hunk 2 above)
Comment (amclelland) [inline config/initializers/assets.rb]: Not sure why rails was complaining about that, I thought `require_tree .` would grab it. But removed this anyway after moving to vendor

**Pull request merged**